kvstore/sql_test.go

```diff
@@ -71,6 +71,22 @@ func TestSQL_Init_db_open_unknown_driver_postgres(t *testing.T) {
 	assert.EqualError(t, err, expectedErr)
 }
 
+func TestSQL_Init_Close_cycle_ok(t *testing.T) {
+	s := Memory{}
```

Comment:
arg!

Reply:
it turns out we cannot run this test because it needs a SQL driver to work (see README.md)